tutorials/cp-starter-integration-cpi-design-iflow/rules.vr

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Please remove this file from your pull request. You should only push this file to Tutorials-Contribution.